components/st-microelectronics/STM32F031F_4-6_Px.supports.stanza

```diff
@@ -119,15 +119,15 @@ public defn make-supports ():
       SPI1_MISO.p => self.PA[6]
     supports SPI1_MOSI:
       SPI1_MOSI.p => self.PA[7]
-    supports spi():
+    supports spi-controller():
       require sdo-pin: SPI1_MOSI
       require sdi-pin: SPI1_MISO
       require sck-pin: SPI1_SCK
       require cs-pin: SPI1_NSS
-      spi().sdo => sdo-pin.p
-      spi().sdi => sdi-pin.p
-      spi().sck => sck-pin.p
-      spi().cs => cs-pin.p
+      spi-controller().cipo => sdo-pin.p
```

Comment:
@electricfungi Isn't it swapped ie cipo = sdo from the mcu perspective or am I reading this wrong?

Reply:
The original mapping from the STM miso pin was swapped - this is unswapping it. We can replace this generated file when we have an updated one.

Reply:
miso --> cipo
m --> c (master --> controller)
s --> p (peripheral) 
it looks like I may have swapped some of the stmicro pins